components/camel-attachments/src/test/java/org/apache/camel/attachment/MulticastAggregationStrategyTest.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.camel.attachment;
```

Comment:
You need license header

Reply:
Thanks. The license header was added in a separate commit and the chery-pick missed. Let me add that one.